InfluxPublisher/c/src/InfluxPublisher.lf

```diff
@@ -117,7 +123,7 @@ reactor SineWave(sample_rate_ms = 100 ms) {
 }
 
 main reactor {
-    publisher = new InfluxLinePublisher()
-    producer = new SineWave()
-    producer.measurement -> publisher.line
+    c_publisher = new InfluxLinePublisher()
```

Comment:
Curious why this change? It makes sense from a naming standpoint, but wanted to know if there was another reason.

Reply:
This is the workaround that we discussed on Wednesday morning. It is this change (described above):
> Change some variable names in the example programs to avoid naming collisions. A more robust and general fix that uses IDs to distinguish between different programs that use the same FQNs is left for a future PR.